app/uk/gov/hmrc/identityverificationquestions/monitoring/auditing/AuditService.scala

```diff
@@ -70,7 +71,13 @@ class AuditService @Inject()(auditConnector: AuditConnector) extends DeviceFinge
 
     val givenAnswer = answerDetails.answer.toString
 
-    val validAnswers: String = questionData.questions.filter(_.questionKey == questionKey).head.answers.mkString(",")
+    val answersList: Seq[QuestionWithAnswers] =
+      questionData.questions.filter(_.questionKey == questionKey)
+
+    val validAnswers: String = {
+      if (answersList.nonEmpty) answersList.head.answers.mkString(",")
+      else s"No Answers Available for the user correlationId: ${questionData.correlationId}. ie, the user has enrolled but has no payments details"
```

Comment:
This feels like the wrong way to handle this, sorry - that else string isn't a valid answer.  Can we think of another approach please?  If there are no valid answers, then the user shouldn't be given the question (i.e. the question key should never be given out for this user)